canonical-web-and-design/vanilla-framework
# Update search in navigation to expanding search

## Done

- Update `run` script
- Keep consistency with search box visible in header throughout all pages
- Update search box to use new expanding search

Fixes #4226 

## QA

- Open [demo](https://vanilla-framework-4228.demos.haus)
- Visit all pages in header
- Expand and close search, check it works as expected.
- Search for something and check it works and gives expected results

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
